boards/sim/sim/sim/Kconfig

```diff
@@ -10,6 +10,10 @@ config EXAMPLES_TOUCHSCREEN_BGCOLOR
 	default 0x007b68ee
 	depends on EXAMPLES_TOUCHSCREEN
 
+config SIM_RPTUN_MASTER
+	bool "Remote Processor Tunneling Role"
+	depends on RPTUN
```

Comment:
Add the default value for this boolean option.

Reply:
> Add the default value for this boolean config.
Done. thank you.